packages/cli/core/README.md

```diff
@@ -31,6 +31,28 @@ USAGE
 
 <!-- usagestop -->
 
+# Configuration
+
+The Coveo CLI is based on oclif and use the following paths:
+
+- **cacheDir** - CLI cache directory
+  - macOS: `~/Library/Caches/mycli`
```

Comment:
Why `mycli`? Shouldn't be `@coveo/cli`

Reply:
oh yeah. I went a bit fast with the copypaste lol